src/uct/ib/ud/base/ud_ep.c

```diff
@@ -544,10 +545,10 @@ ucs_status_t uct_ud_ep_create_connected_common(const uct_ep_params_t *ep_params,
     }
 
     uct_ud_iface_cep_insert_ep(iface, ib_addr, if_addr, path_index, conn_sn, ep);
+    peer_address = (void*)uct_ud_ep_get_peer_address(&ep->conn_match);
```

Comment:
why need cast to void*?

Reply:
`uct_ud_ep_get_peer_address()` returns `const void*`

Reply:
shall we make peer_address const void *?

Reply:
but then we need to cast `peer_address` to `void*` when passing to `uct_ud_iface_unpack_peer_address`

Reply:
maybe use uct_iface_invoke_ops_func(iface, uct_ud_iface_ops_t, ep_get_peer_address, ep) ?

Reply:
done